modules/shared/programs/claude/files/skills/analyzing-da-sessions/scripts/analyze.py

```
@@ -177,85 +615,376 @@ def get_bundle(finding_id: str | None) -> str | None:
+    start = text.rfind("\n###", 0, m.start())
+    if start == -1:
+        start = max(0, m.start() - 800)
+    next_header = text.find("\n###", m.end())
+    if next_header == -1:
+        next_header = min(len(text), m.end() + 1800)
+    return text[start:next_header]
+
+
+def extract_location_identity(block: str) -> str | None:
+    patterns = (
+        r"(?:\*\*)?(?:위치|Location|파일|경로)(?:\*\*)?\s*[:：]\s*`?([^`\n]+)`?",
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)